programs/drift/src/state/user.rs

```diff
@@ -903,6 +903,30 @@ impl PerpPosition {
         }
     }
 
+    pub fn get_be_price(&self) -> DriftResult<i128> {
```

Comment:
breakeven

Reply:
if these are only used for tests, we should move into an impl that is gated by cfg(test) to reduce prod build size, can explain how to do this